lib/t/matches.ex

```diff
@@ -125,7 +128,22 @@ defmodule T.Matches do
         [user_id_1, user_id_2] = Enum.sort(user_ids)
         m = "New match: #{user_id_1} and #{user_id_2}"
         Bot.async_post_message(m)
-        Repo.insert(%Match{user_id_1: user_id_1, user_id_2: user_id_2})
+
+        Multi.new()
+        |> Multi.insert(:match, %Match{user_id_1: user_id_1, user_id_2: user_id_2})
+        |> Multi.insert(:match_event, fn %{match: match} ->
+          %MatchEvent{
```

Comment:
not sure if Multi inside Multi is good here, maybe they could be united via some .insert_all()